src/lib/sql/sql_pipeline.hpp

```diff
@@ -47,7 +49,7 @@ class SQLPipeline : public Noncopyable {
   const SQLQueryPlan& get_query_plan();
 
   // Returns all tasks tht need to be executed for this query.
-  const std::vector<std::shared_ptr<OperatorTask>>& get_tasks();
+  const std::vector<TaskSet>& get_tasks();
```

Comment:
Since we changed the semantics here I would prefer renaming this method to `get_task_sets` (when I read `get_tasks` I assume I get a list of tasks...).